guides/common/modules/proc_performing-a-full-backup.adoc

```diff
@@ -44,13 +44,9 @@ If you want to set a custom directory name, add the `--preserve-directory` optio
 The backup is then stored in the directory you provide in the command line.
 If you use the `--preserve-directory` option, no data is removed if the backup fails.
 
-Note that if you use a local PostgreSQL database, the `postgres` user requires write access to the backup directory.
-
 .Remote databases
```

Comment:
FYI. For DITA migration, only `.Prerequisites` , `.Procedure` and a couple other "approved" title blocks are supported.

As a workaround, you could change these block titles to definition lists: 

`.Remote databases` -> `Remote databases::`